api/user/index.ts

```diff
@@ -5,8 +5,8 @@ export const router = express.Router({ mergeParams: true })
 import { getUserSessionTimeout } from '@hmcts/rpx-xui-node-lib'
 import { getConfigValue } from '../configuration'
 import { SESSION_TIMEOUTS } from '../configuration/references'
+import {exists} from "../lib/util"
```

Comment:
@SivaprakasamSivaprakasam - this ought to be `import { exists } from '../lib/util'` (for consistency of spacing and use of single-quote marks)